docs/providers/aws/guide/functions.md

```diff
@@ -398,6 +402,42 @@ functions:
     architecture: arm64
 ```
 
+## Runtime Management
+
+[Runtime Management](https://docs.aws.amazon.com/lambda/latest/dg/runtimes-update.html) allows for fine-grained control of the runtime being used for a lambda function in the rare event of compatibility issues with a function.
+
+For automatic updates, set `runtimeManagement` to `auto`, and set it to `onFunctionUpdate`
+
+To configure runtime management for all functions, configure `runtimeManagement` at `provider` level as follows:
+
+```yml
+provider:
+  ...
+  runtimeManagement: auto
+```
+
```

Comment:
`runtimeManagement: auto` has no effect, so I don't there's a point in showing such example (we can just write that it defaults to `auto`, which is the AWS default)